lecture2/task2.c

```diff
@@ -0,0 +1,104 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+typedef struct my_ll {
+	int number;
+	char name[16];
+	struct my_ll *next;
+} myll;
+
+int copy_digits(char *inputs,char *outputs)
```

Comment:
if this function always returns just 0, it would make more sense to make it void.